cli/src/main/java/com/devonfw/tools/ide/merge/xml/matcher/IdComputer.java

```diff
@@ -68,7 +71,7 @@ public Element evaluateExpression(Element templateElement, Element workspaceElem
       } else if (length == 0) {
         return null;
       } else {
-        String message = length + " matches found for XPath " + xpathExpr + " in workspace XML at " + XmlMergeSupport.getXPath(workspaceElement, true);
+        String message = length + " matches found for XPath " + xpathExpr + " in workspace XML file `" + workspacePath + "` at " + XmlMergeSupport.getXPath(workspaceElement, true) + " for template file `" + templatePath + "`";
```

Comment:
We typically use single quotes to encapsulate values in log messages (`" bla bla '" + value + "' bla bla "`).
Using backticks here is not wrong but violates homogeneity principle.